packages/file_selector/file_selector_platform_interface/test/method_channel_file_selector_test.dart

```diff
@@ -219,33 +219,62 @@ void main() {
           ],
         );
       });
-      group('#getDirectoryPath', () {
-        test('passes initialDirectory correctly', () async {
-          await plugin.getDirectoryPath(initialDirectory: '/example/directory');
-
-          expect(
-            log,
-            <Matcher>[
-              isMethodCall('getDirectoryPath', arguments: <String, dynamic>{
-                'initialDirectory': '/example/directory',
-                'confirmButtonText': null,
-              }),
-            ],
-          );
-        });
-        test('passes confirmButtonText correctly', () async {
-          await plugin.getDirectoryPath(confirmButtonText: 'Open File');
-
-          expect(
-            log,
-            <Matcher>[
-              isMethodCall('getDirectoryPath', arguments: <String, dynamic>{
-                'initialDirectory': null,
-                'confirmButtonText': 'Open File',
-              }),
-            ],
-          );
-        });
+    });
+    group('#getDirectoryPath', () {
+      test('passes initialDirectory correctly', () async {
+        await plugin.getDirectoryPath(initialDirectory: '/example/directory');
+
+        expect(
+          log,
+          <Matcher>[
+            isMethodCall('getDirectoryPath', arguments: <String, dynamic>{
```

Comment:
I wonder if it might not be worth extracting a utility function `expectMethodCall(log, methodName, {Map<String, dynamic> arguments})`?

Reply:
Like that, will do!